pkg/bootloader/grub.cfg

```diff
@@ -95,13 +95,15 @@ done
 
 insmod part_gpt
 insmod gfxmenu
-loadfont ($root)/grub2/themes/SLE/DejaVuSans-Bold14.pf2
-loadfont ($root)/grub2/themes/SLE/DejaVuSans10.pf2
-loadfont ($root)/grub2/themes/SLE/DejaVuSans12.pf2
-loadfont ($root)/grub2/themes/SLE/ascii.pf2
-insmod png
-set theme=($root)/grub2/themes/SLE/theme.txt
-export theme
+if [ -d "($root)/grub2/themes/SLE" ]; then
+  loadfont ($root)/grub2/themes/SLE/DejaVuSans-Bold14.pf2
+  loadfont ($root)/grub2/themes/SLE/DejaVuSans10.pf2
+  loadfont ($root)/grub2/themes/SLE/DejaVuSans12.pf2
+  loadfont ($root)/grub2/themes/SLE/ascii.pf2
+  insmod png
```

Comment:
I think that `png` module can be loaded anyway, as we may have PNG  image even in TW distro.

Reply:
Nice catch!